File: Modules/ParserObjC/Sources/ParserObjC/ParserData.m
//
//  ParserData.m
//  RSParser
//
//  Created by Brent Simmons on 10/4/17.
//  Copyright © 2017 Ranchero Software, LLC. All rights reserved.
//

#import "ParserData.h"

@implementation ParserData

- (instancetype)initWithURL:(NSString *)url data:(NSData *)data {

	self = [super init];
	if (!self) {
		return nil;
	}

	_url = url;
	_data = data;

	return self;
}

@end

File: Frameworks/RSParser/ParserData.swift
//
//  ParserData.swift
//  RSParser
//
//  Created by Brent Simmons on 6/20/17.
//  Copyright © 2017 Ranchero Software, LLC. All rights reserved.
//

import Foundation

@objc public final class ParserData: NSObject {

	@objc public let url: String
	@objc public let data: Data

	public init(url: String, data: Data) {

		self.url = url
		self.data = data
		super.init()
	}
}
